admin-manual/installation-setup/installation/install-centos.rst

```diff
@@ -68,6 +68,36 @@ Installation instructions
 
    * Archivematica:
 
+   Use the following commands to install the repositories that we are using
```

Comment:
Would suggest making this a bit more explicit:

"While Archivematica 1.7 is in development, please use these commands to install the development repositories:"